addons/bestja_detailed_reports/security/ir.model.access.csv

```diff
@@ -0,0 +1,10 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_detailed_report_users,detailed_report,model_bestja_detailed_report,base.group_user,1,0,0,0
```

Comment:
Why would normal users ever need to have access to this? Also, project managers have to be able to modify reports. And I don't see a reason for a special access for instance_admin. You should merge all your `*_users`, `*_coordinators` and `admin` rules into a one rule for the `bestja_project.managers` group.